src/constants.js

```diff
@@ -2,5 +2,9 @@ export default {
     MESSAGE_ERROR: 0,
     MESSAGE_STATUS: 1,
     MESSAGE_SUCCESS: 2,
-    MESSAGE_WARNING: 3
+    MESSAGE_WARNING: 3,
+    copy: {
+        PAYMENT_REQUEST_DEFAULT: "Thanks, your payment request has been sent. Please take your vouchers to your market representative for them to be sent off.",
```

Comment:
Has this been approved by Steph - i presume "yes" becuase it's copied from elsewhere but check just in case? otherwise needs [xXx]